### PR TITLE
feat: add accessibility links for xpro and mitxonline

### PR DESCRIPTION
**Pull request by asadali145**
## Description

Tickets: https://github.com/mitodl/hq/issues/759 & https://github.com/mitodl/hq/issues/631 & https://github.com/mitodl/hq/issues/682

Here is the related PR that adds the additional footer link https://github.com/mitodl/frontend-component-footer-mitol/pull/9.

## Motivation and Context

Adds a new value for an additional link in the footer. Also, Adds a space in the trademark for the MITx Online

## How Has This Been Tested?

Tested locally only.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (improves on existing behavior)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project. (Did you install and run the pre-commit hooks?)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by pdpinch** [comment]
@asadali145 assuming you are working on https://github.com/mitodl/hq/issues/631 and https://github.com/mitodl/hq/issues/759 you should add links to the description here

**Comment by asadali145** [comment]
> @asadali145 assuming you are working on https://github.com/mitodl/hq/issues/631 and https://github.com/mitodl/hq/issues/759 you should add links to the description here

Yeah, I will finalize the description in a few minutes.

File: src/concourse/pipelines/open_edx/mfe/values.py [hunk 1]
```diff
@@ -115,6 +116,7 @@
         studio_domain="studio.mitxonline.mit.edu",
         support_url="mitxonline.zendesk.com/hc/",
         terms_of_service_url="https://mitxonline.mit.edu/terms-of-service/",
+        accessibility_url="https://accessibility.mit.edu/",
```
Comment (blarghmatey) [inline src/concourse/pipelines/open_edx/mfe/values.py:119]: You will also need to add this as an attribute of the Pydantic model [here](https://github.com/mitodl/ol-infrastructure/blob/main/src/concourse/pipelines/open_edx/mfe/pipeline.py#L37-L57)

Comment (asadali145) [reply src/concourse/pipelines/open_edx/mfe/values.py:119]: Yeah, I did that in my second commit https://github.com/mitodl/ol-infrastructure/pull/1370/commits/3d85506700b5aade6da4cb24a61fdb9ccade576d.

**Comment by asadali145** [comment]
Pre-commit failure is not related to the current changes.

**Review by blarghmatey — APPROVED** [review]

**Comment by asadali145** [comment]
@blarghmatey Could you please verify this comment https://github.com/mitodl/ol-infrastructure/pull/1203#issuecomment-1443718873 and then review these changes?

**Comment by asadali145** [comment]
@blarghmatey Could you please review this PR?

**Review by blarghmatey — APPROVED** [review]

**Pull request merged**